### ISSUE TITLE
Allow see other format hours when using show --totals

### ISSUE THREAD

**Issue by krzysiek-b** [ttscoff/doing#170]
Currently what I see is `Total tracked: days:hours:minutes` which is fine but I would love to see
total hours without days, so days * 24 + hours and an average like
```
Total tracked: 01:02:03 (26h 03 min, 8,12h/day)
```

This could allow me better understand how much do I spend. Possible to be implemented? Can I help?

**Comment by ttscoff** [comment]
No, there's no configuration like that. You can choose text or human for 
the interval format, but the human display is always going to break out 
days. You could probably hack what's there to output it the way you 
want, but it would mean you wouldn't be able to update the code in the 
future. The best solution would be to make that a configuration option 
that could affect the output and then I could merge it into the main 
repo. Explaining all the areas you'd need to update to make this work 
will take some time, but I'll try to get you a better explanation when I 
have some free time. Feel free to dig in the meantime... given nobody 
else has ever requested this I can't consider it a high-priority feature 
:)

-Brett

**Comment by krzysiek-b** [comment]
Hi,

how about doing something like
```
total_hours = total.to_hours
total_minutes = (total_hours % 1 * 60).round
total_days = total_hours / 24
time_span_in_days = ...
avg_hours_per_day = time_span_in_days > 0 ? total_hours / time_span_in_days : total_hours
output += "\n\nTotal tracked: #{total.time_string(format: :clock)} (#{total_hours.floor}h #{total_minutes} min, #{avg_hours_per_day.round(2)}h/day)\n"
```
instead of `Total tracked: #{total.time_string(format: :clock)}`

**Comment by ttscoff** [comment]
Doing 2.1.105 adds this feature. See [release notes](https://github.com/ttscoff/doing/releases/tag/2.1.105).
